.travis.yml

```diff
@@ -27,15 +27,42 @@ jobs:
     - stage: test
       env: TOX_ENV=py34
       python: "3.4"
+      before_install: skip
     - stage: test
       env: TOX_ENV=py35
       python: "3.5"
+      before_install: skip
     - stage: test
       env: TOX_ENV=py36
       python: "3.6"
+      before_install: skip
     - stage: test
       env: TOX_ENV=py37
     - stage: test
       env: TOX_ENV=docs
+      before_install: skip
     - stage: test
       env: TOX_ENV=packaging
+      before_install: skip
+
+    # NOTE: We undo/change all the global Travis CI step definitions to ensure
+    # building of source and wheel distributions is performed in a clean
+    # environment.
+    - stage: release
```

Comment:
you need to override `after_success` here. Check [this](https://github.com/genialis/resolwe/blob/81a3e0854a8b531f8bdda1ae34658defd7598ded/.travis.yml#L116) for details.

Reply:
done